# Designer Genes C

See General Rules, Eye Protection & other Policies on www.soinc.org as they apply to every event.

1. DESCRIPTION: Teams will answer questions, solve problems, and analyze data pertaining to classic, evolutionary, and molecular genetics.

A TEAM OF UP TO: 2			APPROXIMATE TIME: 50 minutes
			
CALCULATOR: Class II

2. EVENT PARAMETERS:
a. Each team may bring writing utensils.
b. Each team may bring two Class II calculators.
c. Each team may bring one (1) 8.5” x 11” sheet of paper that may contain information on both sides in any form and from any source. The sheet of paper may be laminated or placed in a sheet protector to increase durability. Affixed labels, as well as multiple sheets of paper (whether in a single sheet protector or not) are prohibited.

3. THE COMPETITION: This Event will be administered as a written test which may be arranged in stations. Questions will emphasize process skills such as quantitative reasoning, making calculations, analyzing and interpreting experimental results, and drawing evidence-based conclusions. The Event will cover topics listed below without overemphasis on any one particular topic. The list of topics and subtopics should be considered exhaustive.
a. For each of the following topics, participants will be expected to use quantitative reasoning and computational skills, analyze and interpret experimental results, and draw evidence-based conclusions.
i. Mendelian Genetics
(1) Describe Mendel’s Laws of Inheritance and their implications in heredity. Describe inheritance patterns which violate these laws (i.e. linkage, incomplete and codominance, complementation).
(2) Use provided information to construct Punnett Squares of mono-, di- and trihybrid crosses.
(3) Predict genotypes and phenotypes of offspring and compute their likelihood based on Punnett Squares and experimental data using probability rules.
(4) Evaluate pedigrees to predict modes of inheritance (i.e. dominant vs recessive traits, autosomal vs sex-linked traits) and construct pedigrees based on data from a case study.
(5) Understand epistasis and its implications in predicting genetic outcomes.
(6) State and Nationals only: Map two (State) or three (National) genes on a chromosome based on recombination frequency data.
ii. Mitosis and Meiosis
(1) Understand, compare, and contrast the major stages and key structures in mitosis and meiosis.
(2) Describe the effects of nondisjunction and use data from karyotypes to detect chromosomal abnormalities.
(3) State and Nationals Only: Understand somatic recombination in the context of immune cell V(D)J recombination and immunoglobulin class switching.
iii. Population & Evolutionary Genetics
(1) Understand the assumptions and calculations that underlie the Hardy-Weinberg equilibrium and the consequences of violating these assumptions. Be able to make these calculations given data.
(2) Understand and explain the relationships between changing allele frequencies and genetic drift, founder effects, and bottlenecks, migration, and relative fitness.
(3) Explain how additive alleles generate continuous variation and estimate the number of genes contributing to a trait.
(4) Understand the role gene duplication plays in the evolution of the genome and new genes. Be able to identify homologs, orthologs, and paralogs.
(5) Understand how a phylogenetic tree shows evolutionary histories and interpret phylogenies of nuclear, organellar, and extrachromosomal elements. Describe the basic steps of constructing a phylogenetic tree beginning with sequence data input.
(6) State and Nationals only: Understand how heritability is used to explore phenotypic variation in a population. Calculate broad-sense heritability, narrow-sense heritability, and realized heritability.
iv. Molecular Biology of DNA
(1) Be able to identify and explain the components of DNA (i.e. structure of the nucleotides and backbone), directionality of DNA (5’ and 3’ ends), rules in the context of DNA structure and base pairing.
(2) Understand the stages of DNA replication, from assembly of the pre-replication complex to termination, understand in detail the processes that happen at the replication fork.
(3) Understand how DNA polymerases achieve fidelity of replication, and the mechanisms behind DNA proofreading.
(4) Identify and understand the organization of DNA structure, such as plasmids, chromatin complexes, euchromatin and heterochromatin, and chromosomes.
(5) Understanding the processes in which DNA gets damaged and the mechanisms for DNA repair.
(6) Identify, explain, and classify DNA mutations on the DNA level (i.e. chromosomal rearrangements, insertions, deletions, and substitutions), and the potential impact of the resulting protein sequence (i.e. frameshift, silent, missense, and nonsense mutations).
v. Prokaryotic gene expression and regulation
(1) Be able to explain the Central Dogma of Molecular Biology and reverse transcription.
(2) Transcription: Understand and explain initiation, elongation, and termination stages of transcription, the mechanism of RNA polymerase.
(3) Understand the role and general locations of cis- and trans-regulatory elements (i.e. promoters, enhancers, silencers, riboswitches, transcription factors) in the context of a gene and the lac and trp operons.
(4) Translation: Understand and explain the initiation, elongation and termination stages of translation, the mechanism of the ribosome, the role mRNA and tRNA play in translation, and how translation is regulated.
(5) State and Nationals only: Understand protein secretion systems in the context of the Sec and Tat systems.
vi. Technology and Techniques
(1) Polymerase chain reaction (PCR): Describe what occurs and which molecules are involved in the denaturation, annealing, and extension steps of a PCR reaction; Explain the role of temperature in PCR; Identify the components needed for a PCR reaction; Identify experimental questions which could be addressed by PCR.
(2) Sanger sequencing: Describe how the steps and components in Sanger sequencing differ from a standard PCR reaction; Identify experimental questions which could be addressed by Sanger sequencing.
(3) Next-gen (Illumina) vs. third-gen (Nanopore) sequencing: Describe the major platforms used for Next-Gen (Illumina) sequencing. Compare and contrast these methods in terms of how they are performed, data produced, and questions the data addresses.
(4) Molecular cloning: Describe the major steps in cloning a gene of interest to be expressed in a specific organism. Understand the utility of plasmids, restriction enzymes, PCR, Gibson assembly, and blue-white screens in the context of molecular cloning.
(5) Knockout/knockdowns: Understand how knockout, knockdown, and complementation experiments are used to determine gene function. Compare and contrast knockdowns and knockouts and explain why one may be used over the other. State and Nationals only: Describe techniques used to generate knockdown/knockouts (i.e. RNAi, homologous recombination, CRISPR and TALENs).
(6) State and Nationals only: Describe the basic procedure, applications, and limitations of the following techniques: ChIP-seq, Hi-C, RNA-Seq.

4. SCORING:
a. High score wins. Selected questions may be used as tiebreakers.
b. Points will be awarded for quality and accuracy of answers, quality of supporting reasoning, and the use of proper scientific methods.
Recommended Resources: The Science Olympiad Store (store.soinc.org) carries a variety of resources to purchase; other resources are available on the Event Pages at soinc.org.

C
©2026 - C21
